examples/etherDividend.ts

```diff
@@ -1,10 +1,9 @@
 import { BigNumber } from '@0x/utils';
 import { RedundantSubprovider, RPCSubprovider, Web3ProviderEngine } from '@0x/subproviders';
-import { Web3Wrapper } from '@0x/web3-wrapper';
-import { EtherDividendCheckpointEvents, ModuleFactoryContract } from '@polymathnetwork/abi-wrappers/lib/src';
+import { EtherDividendCheckpointEvents } from '@polymathnetwork/abi-wrappers/lib/src';
```

Comment:
This should be imported from the package root

Reply:
There were a few imports mistakes like this. I have fixed them. I am going to move on to the perms task for the next while, seems there are many to update